src/api.js

```diff
@@ -254,7 +254,7 @@ API.prototype.getFees = function () {
     }
   };
 
-  return fetch(this.API_ROOT_URL + 'fees')
+  return fetch(this.API_ROOT_URL + 'mempool/fees?cors=true')
```

Comment:
API_ROOT_URL is `api.blockchain.info` and we're calling from `blockchain.info` so it's a different subdomain. However `cors=true` should not be needed in general.